Exercises-1-Basic/src/exercises/Number7Shape3.java

```diff
@@ -0,0 +1,31 @@
+package exercises;
+
+/*
+  iv)
+	
+
+        *  (5-1)
+       ** (5-2)
+      ***
+    *****
+  *******
+*********
+
+ */
+public class Number7Shape3 {
+
+    public static void main(String[] args) {
+	int number = 1;
+	while (number <= 5) {
+	    for (int j = 1; j <= 5 - number; j++) {
+		System.out.print(" ");
+	    }
+	    for (int i = 1; i <= number; i++)
+		System.out.print("*");
```

Comment:
```suggestion
	    for (int i = 1; i <= number; i++) {
		System.out.print("*");
	    }
```
When using for/while/if you always should to write the inner code inside {}, it is clearer, and won't cause to mistakes.

Reply:
I do automatically formatting so I don't know what are you talking about, can you give me an example?

Reply:
Read my comment again, it shows exactly what to do :-)

Reply:
Done. Please check new file Number5Shape3